src/serac/numerics/mesh_utils.hpp

```diff
@@ -175,28 +170,30 @@ struct InputOptions {
   int par_ref_levels;
 };
 
-}  // namespace mesh
-
 /**
- * @brief Constructs a 3D MFEM mesh of a cuboid
+ * @brief Constructs an MFEM parallel mesh from a set of input options
+ *
+ * @param[in] options The options used to construct the mesh
+ * @param[in] comm The MPI communicator to use with the parallel mesh
  *
- * @param[in] extra_options Cuboid Mesh Options
- * @param[in] MPI_Comm MPI Communicator
  * @return A shared_ptr containing the constructed mesh
  */
-std::shared_ptr<mfem::ParMesh> buildCuboidMesh(serac::mesh::GenerateInputOptions& options,
-                                               const MPI_Comm                     comm = MPI_COMM_WORLD);
+std::shared_ptr<mfem::ParMesh> build(const InputOptions& options, const MPI_Comm comm = MPI_COMM_WORLD);
```

Comment:
`buildSerialMesh`?

Reply:
hmm, that might be confusing because it returns a `ParMesh` - maybe `buildParallelMesh`, though that might be confusing if we also have `refineAndDistribute`.

Reply:
Oh you're right! Whoops!